### PR TITLE
Add `tokenizeBlocks` option and `type` property to block nodes

### PR DESCRIPTION
**Pull request by sosukesuzuki**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by sosukesuzuki** [comment]
I want to move forward, so I'll merge it. If you have any problems please let me know later

**Comment by sosukesuzuki** [comment]
@fisker Could you please make me a collaborator so I can npm publish this package?

File: packages/angular-html-parser/src/index.ts [hunk 1]
```diff
@@ -41,6 +41,10 @@ export interface ParseOptions {
     hasParent: boolean,
     attrs: Array<{prefix: string, name: string, value?: string}>
   ) => void | TagContentType,
+  /**
+   * tokenize blocks (Angular Control Flow Syntax)
+   */
+  tokenizeBlocks?: boolean,
```
Comment (fisker) [inline packages/angular-html-parser/src/index.ts:47]: This options should be named more specific, eg "tokenizeAngularBlocks".

**Comment by fisker** [comment]
> @fisker Could you please make me a collaborator so I can npm publish this package?



> @fisker Could you please make me a collaborator so I can npm publish this package?

You were Invited before.. 
Screenshot_2023-10-29-15-31-33-632_com microsoft emmx canary-edit

Sent you an invitation again.

**Pull request merged**